pkg/services/cloudmigration/cloudmigrationimpl/cloudmigration.go

```diff
@@ -105,11 +105,13 @@ func ProvideService(
 
 	if !cfg.CloudMigration.IsDeveloperMode {
 		// get GMS path from the config
-		domain, err := s.parseCloudMigrationConfig()
-		if err != nil {
-			return nil, fmt.Errorf("config parse error: %w", err)
+		section := cfg.Raw.Section("cloud_migration")
+		domain := section.Key("domain").MustString("")
```

Comment:
This and polling value are always going to be set, could we get the value in `setting_cloudmigration.go`?

Reply:
Forgot we had that, I updated the NewGMSClient method to just take the config struct, and it can pull the config and timeouts it needs from that.